src/session/session_dhandle.c

```diff
@@ -462,15 +462,14 @@ __wt_session_get_btree_ckpt(WT_SESSION_IMPL *session, const char *uri, const cha
      * actually zero in a newer, currently running checkpoint, because then they must have always
      * been zero.)
      *
-     * This scheme relies on the fact we take steps to make sure that the checkpoint wall clock time
-     * does not run backward, and that successive checkpoints are never given the same wall clock
-     * time. Note that we use the write generation to ignore wall clock times from previous database
-     * opens (all such are treated as 0) -- anything from a previous database open can't have been
-     * produced by a currently running checkpoint and can be presumed to match. This is done so we
-     * don't get in trouble if the system clock moves backwards between runs, and also to avoid
-     * possible issues if the checkpoint clock runs forward. (See notes about that in txn_ckpt.c.)
-     * Furthermore, this avoids any confusion potentially caused by older versions not including the
-     * checkpoint time in the snapshot and timestamp metadata.
+     * This scheme relies on the fact that the checkpoint wall clock time always moves forward. Each
+     * checkpoint is given a wall clock time at least one second greater than the previous
+     * checkpoint. Before recovery, we load the time of the last successful checkpoint in the
+     * previous database so we can ensure checkpoint times increase across restarts. This avoids
+     * trouble if the system clock moves backwards between runs, and also avoids possible issues if
+     * the checkpoint clock runs forward. (See notes about that in txn_ckpt.c.) When reading from a
```

Comment:
While you didn't write this part, where in txn_ckpt.c are the referenced notes?

Reply:
I've changed this to cite the function, `txn_checkpoint_establish_time()`, that has these notes.

Reply:
My fault, thanks for fixing it :-)